docs/contributing.md

```diff
@@ -87,6 +87,21 @@ Usage of images and visualizations is encouraged. If you are using an image crea
 
 Feel free to use [emojis](https://docsify.js.org/#/emoji?id=emoji) or [icons](https://icongr.am/fontawesome) where it fits, for example in blockquotes. 
 
+### Prerequisites
+Wiki pages that are subject to future updates, i.e. pages covering active research topics require prerequisites to be added on top of the pages:
+
+* Warning message artifact using the following format:
+> [!WARNING]
+> This document covers an active area of research, may be outdated at time of reading and subject to future updates, as the design space around (*active research topic, e.g. Inclusion Lists*) evolves.
+
+* Roadmap tracker artifact using the following format:
```

Comment:
```suggestion
#### Rodmap format
Roadmap tracker artifact using the following format:
```

Reply:
One thing about roadmap tracker: if we go for the proposed changes, it's not clear imo if the tracker should be added to all pages or just to research pages([acc. to this discussion](https://github.com/eth-protocol-fellows/protocol-studies/issues/179#issuecomment-2056404495)). That's why in the initial commit, both artifacts were presented under `research` pages